### PR TITLE
fix(schema): Make label & description optional for hidden actions

### PR DESCRIPTION
**Pull request by xavdid**
Noticed the other day that hidden triggers needed label/description. This loosens that requirement. 

The one thing I still need to test is that the editor plays nicely with this. I'll do that before I merge it.

### CODE REVIEW

File: packages/schema/lib/functional-constraints/labelWhenVisible.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+  for (const actionType of actionTypes) {
+    const group = definition[actionType] || {};
+    _.each(group, (action, key) => {
+      const { display } = action;
+      if (!display.hidden && !(display.label && display.description)) {
+        errors.push(
+          new jsonschema.ValidationError(
+            `visible actions must have a label and description`,
+            action,
+            `/${_.capitalize(actionType)}Schema`,
```
Comment (eliangcs) [inline packages/schema/lib/functional-constraints/labelWhenVisible.js]: Should we link to `BasicDisplaySchema` instead?

File: packages/schema/lib/functional-constraints/labelWhenVisible.js [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+            `instance.${key}.key`,
```
Comment (eliangcs) [inline packages/schema/lib/functional-constraints/labelWhenVisible.js]: Ideally, I think the path should be `instance.${key}.display.label` or `instance.${key}.display.description` to be more accurate, but I'm ok with `instance.${key}.display` too.

Comment (xavdid) [reply packages/schema/lib/functional-constraints/labelWhenVisible.js]: went with `instance.creates.foo.display`, which is the path to where the dev will need to fix the issue. Good catch!

**Comment by xavdid** [comment]
confirmed locally this works without issue in the editor!

**Pull request merged**